test/metadata-retrieval/command.go

```diff
@@ -0,0 +1,67 @@
+package metadataretrieval
+
+import (
+	"os"
+	"os/exec"
+	"syscall"
+
+	"github.com/src-d/regression-core"
+)
+
+// Command wraps a metadata-retrieval server instance.
+type Command struct {
+	cmd    *exec.Cmd
+	binary string
+	orgs   string
+	dir    string
+}
+
+// NewCommand creates a new metadata-retrieval command struct.
+func NewCommand(binary, orgs string) *Command {
+	return &Command{
+		cmd:    new(exec.Cmd),
+		binary: binary,
+		orgs:   orgs,
+	}
+}
+
+// Run runs metadata-retrieval util to discover and download organizations
+func (c *Command) Run(envs map[string]string) error {
+	// TODO(kyrcha): Probably not needed since there is no download of data
```

Comment:
indeed, the only place I see where dir is used is it's own cleanup :)
we can try to remove it